modules/consentManagement.js

```diff
@@ -156,19 +167,21 @@ function lookupIabConsent({onSuccess, onError}) {
   }
 }
 
+let timer = null;
```

Comment:
the timer for `actionTimeout` should be kept separate from the original `timeout` I think, to keep the same semantics for the latter.